src/utils.ts

```diff
@@ -29,23 +25,3 @@ export async function findCurrentPr(): Promise<any | undefined> {
     });
     return result.data.find(pr => pr.state === "open" && github.context.payload.ref === `refs/heads/${pr.head.ref}`);
 }
-
-export function getArtifactName(defaultName?: string): string {
```

Comment:
Do we no longer need this because shared artifacts changed outputs/inputs between jobs ?
In addition to that, I guess we are locking down the sonar step in the workflow so that only one instance runs it in the matrix (instead of relying on shared artifacts) ?

Reply:
That's right 👍 After further consideration the shared artifacts seemed unnecessary.

Their purpose was (a) to pass data between multiple jobs in the same workflow or (b) as a "lock" to ensure that a step runs only once in a matrix. We can accomplish both things with other means - for (a) we can use job outputs, and for (b) we can use an `if` condition to lock down a step as you mentioned.